.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,42 @@
+| Q | A
+| --- | ---
+| Bug fix? | [yes|no]
+| New feature? | [yes|no]
+| BC breaks? | [yes|no]
+| Deprecations? | [yes|no]
+| Tests pass? | [yes|no]
+| Fixed tickets | [Example: fixes #123, fixes #456]
+| Related PRs | [Example: #345, #789]
+| License | MIT
+| Documentation PR | [Example: sulu-io/sulu-docs#123 - leave empty if not needed]
+
+#### TODO
```

Comment:
I would rename this to "To Do" and move this section to the very end. It's the least important section of all.